.gas-snapshot

```diff
@@ -98,5 +98,5 @@ RestrictedNORI__validateSchedule_reverts:test_startTimeZero() (gas: 3350)
 RestrictedNORI_initialize:test() (gas: 21825)
 RestrictedNORI_scheduleExists:test() (gas: 15027)
 RestrictedNORI_scheduleExists:test_doesntExist() (gas: 15006)
-Checkout_buyingFromOneRemoval:test() (gas: 810319)
-Checkout_buyingFromTenRemovals:test() (gas: 3082458)
+Checkout_buyingFromOneRemoval:test() (gas: 817084)
+Checkout_buyingFromTenRemovals:test() (gas: 3102580)
```

Comment:
20,000 gas more expensive :skull:

Reply:
I think a standard ETH transfer is 21,000 gas, so this is more expensive by only about 1 standard transaction